run_haybaler.sh

```diff
@@ -61,3 +61,10 @@ fi
 python3 haybaler.py -i "$input_files" -p . -op $outputDir  -o haybaler.csv
 # for pipeline testing only!!
 #python3 haybaler.py -i "$input_files" -p . -op $outputDir  -o haybaler.csv --readcount_limit 1 --rpmm_limit 10
+
+# Move log file into log directory
+if [[ ! -d $outputDir/logs ]]
+    then
+    mkdir $outputDir/logs
```

Comment:
You can shorten this whole thing to `mkdir -p bla/logs` - I only learned this recently

Reply:
Thanks, I changed it for the other (not log directories) in run `run_heattrees.sh` as well